pyFAST/fastfarm/FASTFarmCaseCreation.py

```diff
@@ -96,7 +102,9 @@ def __init__(self,
         wts: dictionary
             Wind farm layout and turbine parameters in dictionary form
         tmax: scalar
-            Max simulation time given in seconds
+            Wind field time given in seconds
+        tsim: integer
+            Multiple of tmax to repeat the wind field
```

Comment:
This seems to be something that only applied to the Mann box? I'm not sure as I haven't used Mann extensively. Changing the description of `tmax` to "wind field time [...]` is misleading because this also the time used on FAST.Farm.

Reply:
This is also an option in TurbSim. Your simulation time can be multiple of your windfield time, this way you do not create big windfields.